html/templates/data.html

```diff
@@ -1,41 +1,667 @@
-{% extends "base.html" %}
-{% set active = "data" %}
-
-{% block content %}
+{% extends "base.html" %} {% set active = "data" %} {% block content %}
+<style>
```

Comment:
This style should go into a separate .css file. Add it in static/css, e.g. custom.css, and include it in the base template or overwrite the header in the base template